github/github_beta.go

```diff
@@ -0,0 +1,95 @@
+package github
```

Comment:
It's now merged in `go-github`, but since there's no release yet, I could update go.mod to use a tag instead of the last release if you prefer (so this file can be removed)